images/bootstrap/Makefile

```diff
@@ -18,8 +18,8 @@ TAG = $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 all: build
 
 build:
-	bazel build //images/bootstrap/barnacle:barnacle
-	cp ./../../bazel-bin/images/bootstrap/barnacle/linux_amd64_stripped/barnacle ./barnacle/barnacle
+	bazel build //images/bootstrap/barnacle:barnacle --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
+	cp ./../../bazel-bin/images/bootstrap/barnacle/linux_amd64_pure_stripped/barnacle ./barnacle/barnacle
```

Comment:
I wonder if at some point we want to use something like https://github.com/bazelbuild/rules_go/blob/213d5fcd0853a51f970a2cdd4c984b411d5bbf79/tests/examples/executable_name/BUILD.bazel#L8-L13 to normalize the output name.